repository/c.json

```diff
@@ -3141,6 +3141,20 @@
 				}
 			]
 		},
+		{
+			"name": "CompareBuff",
+			"details": "https://github.com/nexional/CompareBuff",
```

Comment:
Since the info can be retrieved simply from this GitHub repo url, you can remove "name", "readme" and "issues".